src/components/selectList/selectList.tsx

```diff
@@ -1,9 +1,9 @@
-import { useCallback, useState } from 'react';
-import styled, { css } from 'styled-components';
-import { DestinationNetworkEnum, useStore } from '../../helpers';
-import { fontSize, pxToRem, spacing } from '../../styles';
-import { IconButton } from '../iconButton/iconButton';
-import { TextField } from '../textField/textField';
+import {useCallback, useState} from 'react';
+import styled, {css} from 'styled-components';
+import {DestinationNetworkEnum, useStore} from '../../helpers';
+import {fontSize, pxToRem, spacing} from '../../styles';
+import {IconButton} from '../iconButton/iconButton';
+import {TextField} from '../textField/textField';
```

Comment:
line 1 to 6 should be fixed => there should be space between curly brackets and the name ... just saving the file probably fixes it already

Reply:
i think it`s not eslint

i guess i should change it in webstorm settings 
cos i changed PC and clear settings